coreneuron/network/netcvode.cpp

```diff
@@ -232,7 +236,7 @@ void NetCvode::clear_events() {
         d.unreffed_event_cnt_ = 0;
         d.inter_thread_events_.clear();
         d.tqe_->nshift_ = -1;
-        d.tqe_->shift_bin(nrn_threads->_t);
+        d.tqe_->shift_bin(nrn_threads->_t - 0.5 * nrn_threads->_dt);
```

Comment:
There is a similar statement in the following function I did not change. A specific test would have to be written to see if there could be an issue with file mode demonstrating the need for an equivalent change.

Reply:
The corresponding change was made in NetCvode::init_events.  In retrospect, it is likely that BinQ is not supported for ```--restore``` since there is no mention of ```shift_bin``` in the ```nrn_checkpoint``` files. (Analogous to ```shift_bin``` in nrn/src/nrniv/bbsavestate.cpp)